### ISSUE TITLE
kcov issue with braces in if check

### ISSUE THREAD

**Issue by Mark-E-Hamilton** [SimonKagstrom/kcov#63]
Hi,again,

I hope I'm not keeping you from other (paying) work. ;)

I have another test case that looks like this:
````
#!/bin/bash
echo "Hello"
function some_test
    {
    test "${1}" == "--quiet" &&
        { quiet=${1} ; shift ; }
    }
````
The kcov report correctly shows the echo as green (covered) and the test statement as red (uncovered.) However, it shows the brace delimited quiet statement as while (uncovered) and the total coverage as 50%.

If I change the quiet line to either of these 
````
        {
        quiet=${1} ; shift ;
        }
````
````
        {
        quiet=${1}
        shift
        }
````
kcov correctly reports them all as uncovered.

**Comment by SimonKagstrom** [comment]
No problem, it's evening here and I'm free now! The amount of time I can spend on kcov will vary though!

I'll add a regression test to catch this. Should be solvable, although it seems there's no end to bash syntax creativeness :-)

**Comment by SimonKagstrom** [comment]
Fixed with 9e544e9

**Comment by Mark-E-Hamilton** [comment]
Yeah, I do tend to use bash creatively! ;)

Thanks for the quick response.
